packages/connect-react/src/hooks/form-context.tsx

```diff
@@ -395,6 +436,98 @@ export const FormContextProvider = <T extends ConfigurableProps>({
     checkPropsNeedConfiguring()
   };
 
+  const handleSdkErrors = (o: unknown[] | unknown | undefined) => {
+    let newErrors = [
+      ...sdkErrors,
+    ]
+    if (!o) return
```

Comment:
Return earlier to avoid copying an array that we won't use.
```suggestion
    if (!o) return
    let newErrors = [
      ...sdkErrors,
    ]
```